students/dabeen/westagram/users/views.py

```diff
@@ -1,3 +1,77 @@
 from django.shortcuts import render
 
+import json
+import bcrypt
+import re
+import jwt
+from django.http import JsonResponse
+from django.views import View
+from users.models import User
+
 # Create your views here.
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email = data['email']
+            password = data['password']
+
+            # Return 400 error when using an existing email.
```

Comment:
주석 없이도 충분히 가독성 좋은 코드를 작성해주셔서, 주석은 필요 없을 것 같아요~
```suggestion
            # Return 400 error when using an existing email.
```